### PR TITLE
Skip the termination of ec2_stop instances

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [ ] enhancement
- [ ] documentation
- [x] dependencies

### CODE REVIEW

File: cloud_governance/policy/aws/ec2_stop.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def __fetch_stop_instance(self, instance_days: int, delete_instance_days: int, s
                         delta_charge = self.get_ebs_cost(resource=block_device_mappings[instance_id], resource_type='ec2', resource_hours=(self.DAILY_HOURS * (self.DELETE_INSTANCE_DAYS-self.DAYS_TO_NOTIFY_ADMINS)))
                         stop_cost = self.get_ebs_cost(resource=block_device_mappings[instance_id], resource_type='ec2', resource_hours=(self.DAILY_HOURS * days))
                         self.__trigger_mail(tags=tags, stopped_time=stopped_time, days=days, resource_id=instance_id, image_id=ami_id, ec2_type=ec2_types[instance_id], instance_id=instance_id, message_type='delete', stop_cost=stop_cost, delta_charge=delta_charge)
-                        self._ec2_client.terminate_instances(InstanceIds=[instance_id])
+                        # self._ec2_client.terminate_instances(InstanceIds=[instance_id])
```
Comment (ebattat) [inline cloud_governance/policy/aws/ec2_stop.py:95]: Are we going to do snapshot backup ?

Comment (athiruma) [reply cloud_governance/policy/aws/ec2_stop.py]: yes, commented ami too.

File: cloud_governance/policy/aws/ec2_stop.py [hunk 2]
```diff
@@ -87,15 +87,15 @@ def __fetch_stop_instance(self, instance_days: int, delete_instance_days: int, s
                     tag_specifications = [{'ResourceType': 'image', 'Tags': tags}]
                     if sign == ge:
                         tag_specifications.append({'ResourceType': 'snapshot', 'Tags': tags})
-                    try:
```
Comment (ebattat) [inline cloud_governance/policy/aws/ec2_stop.py:90]: Pls add also comment why we comment out this part, so we will know the reason.
"Comment out termination code to reduce the risk of unintended terminations"

Comment (athiruma) [reply cloud_governance/policy/aws/ec2_stop.py:90]: done

**Review by ebattat — APPROVED** [review]

**Pull request merged**